ShellScript/Bash.sublime-syntax

```diff
@@ -157,50 +157,50 @@ contexts:
   control:
     - match: \b(if){{keyword_boundary_end}}(?:\s*(!))?
       captures:
-        1: keyword.control.if.begin.shell
+        1: keyword.control.conditional.if.shell
         2: keyword.operator.logical.shell
     - match: \bthen\b
-      scope: keyword.control.if.then.shell
+      scope: keyword.control.conditional.then.shell
       pop: true
     - match: \b(elif){{keyword_boundary_end}}(?:\s*(!))?
       captures:
-        1: keyword.control.if.elif.shell
+        1: keyword.control.conditional.elif.shell
         2: keyword.operator.logical.shell
     - match: \bfi{{keyword_boundary_end}}
-      scope: keyword.control.if.end.shell
+      scope: keyword.control.conditional.end.shell
       set: [cmd-post, cmd-args]
     - match: \belse{{keyword_boundary_end}}
-      scope: keyword.control.if.else.shell
+      scope: keyword.control.conditional.else.shell
       pop: true
     - match: \bfor{{keyword_boundary_end}}
-      scope: keyword.control.for.shell
+      scope: keyword.control.loop.for.shell
       set: [cmd-post, for-args]
     - match: \bdo{{keyword_boundary_end}}
-      scope: keyword.control.do.shell
+      scope: keyword.control.loop.do.shell
       pop: true
     - match: \bdone{{keyword_boundary_end}}
-      scope: keyword.control.done.shell
+      scope: keyword.control.loop.done.shell
```

Comment:
Hm, I never though about closing keywords of loop structures in languages that don't use significant whitespace or curlies.

What do you think about `keyword.control.loop.end`?